File: problemas/iniciante/1174/1174.js
process.stdin.setEncoding('utf-8');

let input = '';

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    const Vetor = input.trim().split('\n').map(Number);
    for (let i = 0; i <= Vetor.length; ++i) {
        const valor = parseFloat(Vetor[i]);
        if (valor <= 10) {
            console.log(`A[${i}] = ${valor.toFixed(1)}`);
        }
    }
});

File: problemas/iniciante/1174/1174.py
def main():
    vetor = []
    for entrada in range(0, 100):
        try:
            entrada = float(input())
            vetor.append(entrada)
        except EOFError:
            break

    for i in range(0, len(vetor)):
        X = vetor[i]
        if X <= 10:
            print(f"A[{i}] = {X}")
if __name__ == "__main__":
    main()
